### ISSUE TITLE
install_requires question

### ISSUE THREAD

**Issue by Hunter0904** [analysiscenter/cardio#38]
HI all :
   When I am installing  this project ,the install_requires needs scikit-learn=='0.19.1' 
   but  scikit-learn 0.19.1  I can't install correctly
   I think it's too old version?
   I user annaconda +python3.7
  how could I do to work with this project~ thanks  a lot 
![image](https://user-images.githubusercontent.com/67938741/86700873-16242c80-c044-11ea-8412-c25c29a10de5.png)

**Comment by dpodvyaznikov** [comment]
Hi, @Hunter0904 !

It seems that the problem is in conflict of `numpy` and 'scikit-learn` version. I was unable to reproduce this issue on Ubuntu 18.04, so it seems that your OS also has something to do with it.

We had to fix `scikit-learn` version to keep segmentation pipelines working because `scikit-learn` had changes in its' API that were not supported by `hmmlearn` at the moment.

I did some tests and segmentation seems to work fine for me on Ubuntu 18.04 with `hmmlearn==0.2.2`, `numpy==1.17.4` and `scikit-learn==0.21.3`. 

So, I'd recommend to install latest versions of the libraries and try running your task.
